gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -72,48 +72,52 @@ internal class AvatarPickerViewModel(
             AvatarPickerEvent.DownloadManagerDisabledDialogDismissed -> hideDownloadManagerAlert()
             is AvatarPickerEvent.AvatarDeleteSelected -> deleteAvatar(event.avatarId)
             AvatarPickerEvent.AvatarDeleteAlertDismissed -> hideNonSelectedAvatarAlert()
-            is AvatarPickerEvent.AvatarRatingSelected -> updateAvatar(event.avatarId, event.rating)
+            is AvatarPickerEvent.AvatarRatingSelected -> updateAvatarRating(event.avatarId, event.rating)
+            is AvatarPickerEvent.AvatarAltTextTapped -> launchAltTextEditor(event.avatarId)
         }
     }
 
-    private fun updateAvatar(avatarId: String, rating: Avatar.Rating? = null, altText: String? = null) {
+    private fun updateAvatarRating(avatarId: String, rating: Avatar.Rating? = null) {
```

Comment:
The AvatarPicker modifies only the `Rating` so there is no need for the generic method that allow the `altText` update. I've remove/renamed all the related code.